ide/libs.lucene/nbproject/project.properties

```diff
@@ -19,3 +19,8 @@ is.autoload=true
 javac.source=1.8
 javac.compilerargs=-Xlint -Xlint:-serial
 release.external/lucene-core-3.6.2.jar=modules/ext/lucene-core-3.6.2.jar
+
+# This is an very old library, the complete dependencies were never explored.
+# Since subpackage-s are working in new sigtest version, generation of the
+# signature file fails. Disabling that now.
```

Comment:
Disabling is fine, the comment though does not match the reason (from my POV). The reported error is:

```
/home/matthias/src/netbeans/nbbuild/templates/projectized.xml:765: 
java.lang.AssertionError: org.apache.lucene.search.CachingWrapperFilter$FilterCache<org.apache.lucene.search.DocIdSet>
	at com.sun.tdk.signaturetest.loaders.BinaryClassDescrLoader.load(BinaryClassDescrLoader.java:225)
	at com.sun.tdk.signaturetest.core.ClassHierarchyImpl.load(ClassHierarchyImpl.java:180)
	at com.sun.tdk.signaturetest.core.ClassHierarchyImpl.getClassInfo(ClassHierarchyImpl.java:292)
	at com.sun.tdk.signaturetest.core.ClassHierarchyImpl.isInterface(ClassHierarchyImpl.java:272)
	at com.sun.tdk.signaturetest.core.ClassCorrector.fixType(ClassCorrector.java:315)
	at com.sun.tdk.signaturetest.core.ClassCorrector.fixMethods(ClassCorrector.java:298)
```

So I would say this is the same as in other cases: sigtest fails to parse the API classes.